test/test.php

```diff
@@ -39,7 +39,8 @@ public function isOk() {
   'pdo_mysql',
   'simplexml',
   'xsl',
-  'xdebug'
+  'xdebug',
+  'IonCube Loader'
```

Comment:
Add a trailing `,` to avoid having to modify 2 lines for a 1 line change in the future